xrpl-wasm-std/src/core/current_tx/mod.rs

```diff
@@ -70,6 +70,7 @@ use crate::host::error_codes::{
 use crate::host::{Result, get_tx_field, to_non_optional};
 
 pub mod escrow_finish;
+pub mod contract_call;
```

Comment:
I fee like this can be done better? Or do we really need to create a new one for each function that might be the "current_tx"?